Linux Build/Install/Test CI, Docs, and Coding Standard Script

kencasimiro opened the pull request:
- Documentation completed for ReadtheDocs
- Coding Standard GitHub Actions completed
- Linux build/install/test CI completed

kencasimiro committed:
Added coding standard script

kencasimiro committed:
Create clang-format-fix.yml

github-actions[bot] committed:
Committing clang-format changes

kencasimiro committed:
Create clang-format-check.yml

kencasimiro committed:
Create linux

kencasimiro committed:
Rename linux to linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Sphinx quick-start

kencasimiro committed:
Merge branch 'develop' of https://github.com/kencasimiro/vol-async into develop

kencasimiro committed:
Added theme for sphinx

kencasimiro committed:
Create outline for doc

kencasimiro committed:
Finalize subcategories

kencasimiro committed:
Finished Background subcategory

kencasimiro committed:
Finished Introduction Section

kencasimiro committed:
Added other categories

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Completed copyright section

kencasimiro committed:
Completed license section

kencasimiro committed:
Merge branch 'develop' of https://github.com/kencasimiro/vol-async into develop

kencasimiro committed:
Fixed code blocks and subcategories

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Updating fork

kencasimiro committed:
Merge pull request #2 from hpc-io/develop

Updating fork

github-actions[bot] committed:
Committing clang-format changes

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Merge branch 'develop' of https://github.com/hpc-io/vol-async into develop

kencasimiro committed:
Update fork

kencasimiro committed:
Updating fork

github-actions[bot] committed:
Committing clang-format changes

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update async_test_serial_event_set_error_stack.c

kencasimiro committed:
Updating fork

github-actions[bot] committed:
Committing clang-format changes

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Merge branch 'hpc-io:develop' into develop

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Merge branch 'hpc-io:develop' into develop

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update linux.yml

kencasimiro committed:
Update fork

kencasimiro committed:
Merge branch 'develop' of https://github.com/kencasimiro/vol-async into develop

kencasimiro committed:
Update linux.yml

github-actions[bot] committed:
Committing clang-format changes

kencasimiro committed:
Update fork

kencasimiro committed:
Update

github-actions[bot] committed:
Committing clang-format changes

kencasimiro committed:
Update linux.yml

houjun commented on .clang-format:
```diff
@@ -0,0 +1,58 @@
+  - Regex:           '.*'
+    Priority:        0
+    SortPriority:    0
+  - Regex:           '^PDC*.*'
```
This is not needed.

houjun reviewed (commented):
@kencasimiro, I just formatted the codes under src and tests with your clang-format file, can you rebase this pull request on the latest develop branch?

kencasimiro committed:
Update .clang-format

Removed the following:
  - Regex:           '.*'
    Priority:        0
    SortPriority:    0
  - Regex:           '^PDC*.*'
    Priority:        1
    SortPriority:    0

kencasimiro committed:
Merge branch 'hpc-io:develop' into develop

houjun reviewed (approved):
Looks great, thanks @kencasimiro !
